### DISCUSSION TITLE
Neo needs a reliable plug

### DISCUSSION THREAD

**Discussion by KylePyleBoFyle** [community/community#81208] [Other Feature Feedback, Questions, & Ideas]
I need help with stuff and fast quick to the point.

**Comment by Talha-Hamza** [comment]
I highly recommend Github Copilot

**Comment by samus-aran** [comment]
Hi @KylePyleBoFyle ! We hope you found what you were looking for. ✨

We're locking this discussion because it has not had recent activity and/or other members have asked for more information to assist you but received no response. Thank you for helping us maintain a productive and tidy community for all our members.
